### ISSUE TITLE
Range 7 AR scenarios, unclear what they do

### ISSUE THREAD

**Issue by 132nd-Entropy** [132nd-vWing/ATRM_Brief#33]
from the brief:
Admin
Start AR scenario: “Start Dynamic AR Scenario”
“Activate AR Targets at Range 7” + “Activate Anti Air at Range 7”

its unclear for the user what those do, or what the difference is

**Comment by 132nd-Entropy** [comment]
friendly bump

**Comment by 132ndNeck** [comment]
I suggest we remove this:
“Activate AR Targets at Range 7” + “Activate Anti Air at Range 7”
If I remember correctly, it refers to a basic AR setup, which not got used very often.  (and is not documentet anywhere with a brief).
I think it refers to this one: https://github.com/132nd-vWing/ATRM/issues/20
 We put it in the .miz at the start, before we developed own scenarios for the various ranges. So it can be deleted both from the .miz/F10 menu and the briefing I think.


The “Start Dynamic AR Scenario” refers to this one: https://132nd-vwing.github.io/ATRM_Brief/Pages/R7_AR_TASK.html

**Comment by 132nd-Entropy** [comment]
yeah I just looked up the script, youre right. since we have the basic AR setup on other ranges, we can take it out here (or, leave it in, always good to have options after all, and create a new entry in the brief, Im happy either way.
